app/src/test/kotlin/compiler/e2e/CorrectPrograms.kt

```diff
@@ -0,0 +1,182 @@
+package compiler.e2e
+
+import kotlin.test.Test
+
+class CorrectPrograms {
+    @Test
+    fun `test identifiers`() {
+        E2eAsserter.assertProgramCorrect(
+            """
+            zm n: Liczba = 10234
+            zm camelCaseIdentifier: Liczba = 10234
+            zm snake_case_identifier: Liczba = 10234
+            zm ident1f13r_w1tH_n00mb3r5: Liczba = 10234
+            """
+        )
+    }
+
+    @Test
+    fun `test types and literals`() {
+        E2eAsserter.assertProgramCorrect(
+            """
+            zm n: Liczba = 10234
+            n = 0
+            n = -0
+            n = -56789
+            n = +56789
```

Comment:
This breaks as there is no unary plus operator in the grammar. You can add it to the grammar or remove this line.

Reply:
I just removed the plus, we don't need it